src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.ts

```diff
@@ -17,15 +18,22 @@ export class EcoNewsDetailComponent implements OnInit, OnDestroy {
   private newsItem: EcoNewsModel;
   private newsImage: string;
   private images = singleNewsImages;
+  private userId: number;
```

Comment:
You have those variables in template, so - they are data bound properties!
**Data bound properties must also be public** look here - https://angular.io/guide/aot-compiler

Reply:
Fixed